From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sun, 26 Aug 2012 11:13:00 -0400
Subject: keyboard: Add launch terminal shortcut

The keyboard shortcut for "Launch terminal" was dropped in the
gconf>gsettings migration. Restore it. (LP: #1010558, LP: #1040081)

This could return upstream if GNOME integrates xdg-terminal-exec support
---
 panels/keyboard/01-launchers.xml.in | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/panels/keyboard/01-launchers.xml.in b/panels/keyboard/01-launchers.xml.in
index cdeb4bd..cc5d9f3 100644
--- a/panels/keyboard/01-launchers.xml.in
+++ b/panels/keyboard/01-launchers.xml.in
@@ -9,6 +9,8 @@
 
 	<KeyListEntry name="email" description="Launch email client"/>
 
+	<KeyListEntry name="terminal" description="Launch terminal"/>
+
 	<KeyListEntry name="www" description="Launch web browser"/>
 
 	<KeyListEntry name="home" description="Home folder"/>
